lib/uplink/ext/c/tests/common_test.c

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+#include "unity.h"
+#include "../../uplink-cgo.h"
+
+void TestGetIDVersion(void)
+{
+    char *_err = "";
+    char **err = &_err;
+    uint8_t id_version_number = 0;
+
+    IDVersion_t id_version = GetIDVersion(id_version_number, err);
```

Comment:
Is PascalCase standard code writing in C?

Reply:
PascalCase for types seems to be a c convention; however, we currently are also using it for cgo exported functions and some helper functions. This is not conventional and we plan to change those to snake_case, which is.

Reply:
Wasn't PascalCase only for C++... although, I've seen PascalCase C code as well, but not very commonly.